packages/block-editor/src/components/rich-text/use-paste-handler.js

```diff
@@ -58,13 +58,35 @@ export function usePasteHandler( props ) {
 			const isInternal =
 				event.clipboardData.getData( 'rich-text' ) === 'true';
 
+			function pasteInline( content ) {
+				const transformed = formatTypes.reduce(
+					( accumlator, { __unstablePasteRule } ) => {
```

Comment:
While we're here, we can fix the typo:

```suggestion
					( accumulator, { __unstablePasteRule } ) => {
```

Reply:
What do you mean, you didn't know about the abbreviation accum'lator? 😄